compiler/onecc-docker/onecc-docker

```diff
@@ -32,17 +32,18 @@ class RequestHandler:
 
     def make_request(self, url):
         try:
-            response = requests.get(url, headers=self.headers)
+            response = requests.get(url, headers=self.headers, timeout=self.timeout)
             response.raise_for_status()
             return response
         except requests.RequestException as e:
             raise SystemExit('[onecc-docker] error: {}'.format(e))
 
 
+# 5 sec timeout is set based on github.com/Samsung/ONE/issues/11134
 def _request_recent_version(token=None):
     response = RequestHandler(
         token,
-        timeout=100).make_request(url="https://api.github.com/repos/Samsung/ONE/releases")
+        timeout=5).make_request(url="https://api.github.com/repos/Samsung/ONE/releases")
```

Comment:
NOTE: this 100 sec timeout was set here but not being used in the request above.